src/doc/gui_manual/Operators/OperatorTypes/PICS/index.rst

```diff
@@ -0,0 +1,268 @@
+operator. The user selects a series of seed locations where curves are
+generated, which are then visualized and analyzed.
+
+The PICS allows for the computation of Lagrangian Coherent Structures (LCS)
+using a variety techniques developed by George Haller and his group at ETH
+Zürich. Haller proposed that the ridges of Finite Time Lyapunov Exponents
+(FTLE) are indicators of hyperbolic LCS (repelling and attracting). In two
+dimensional flows, hyperbolic LCS's turn out to be stationary curves of the
+averaged material shear while elliptic LCS's are stationary curves of the
+averaged strain. The LCS operator coupled with the Integral Curve (Hyperbolic
+LCS) and Limit Cycle (Elliptical LCS) operators provide a mechanism to
+calculate these structures and have been developed in cooperation between the
+VisIt PICS team and the Haller Group.
+
+INCLUDE IMAGE of elliptical and hyperbolic LCS
+
+For complete details on computing elliptical LCS (closed orbits) and hyperbolic
+LCS (stretch and shrink lines) see the following paper: `K. Onu, F. Huhn, & G.
+Haller, LCS Tool: A Computational platform for Lagrangian coherent structures,
+J. of Computational Science, 7 (2015) 26-36
+<http://georgehaller.com/reprints/lcstool.pdf>`_
+
+.. toctree::
+    :maxdepth: 1
+
+    IntegralCurve_operator
+    LCS_operator
+    LimitCycle_operator
+    Poincare_operator
+
+Parameters
+----------
+
+Common to all PICS operators is a four tab GUI: Source, Integration,
+Appearance, and Advanced (the Poincare operator also has an Analysis tab).
+These tabs contain many functions that are common across all four operators.
+The following is a description of those common features. Please note: on a mac
+system, the contents of the Source and Integration tabs are combined into a
+single Integration tab.
+
+
+Source
+^^^^^^
+
+The set of points to seed the integral curves in the operator. See specific
+operator for details.
+
+Field
+"""""
+
+Sets the field type so that the native elements are used when interpolating the
+vector fields. Each operator provides the following options:
+
+Default
+    Use VisIt's native VTK mesh structure performing linear interpolation on
+    the vector field.
+    
+Flash
+    Evaluates the velocity field via the Lorentz force. Parameters are:
+
+    * Constant - A constant multiple applied to the velocity.
+    * Velocity - When combined with Leap-Frog integration, this sets the
+    initial velocity used in the integration.
+   
+M3D-C1 2D
+    Evaluates the 3D magnetic field via a 2D poloidal 6th order polynomial.
+    Parameters are:
+
+    * Constant - A constant multiple applied to the perturbed part of the
+      field.
+   
+M3D-C1 3D
+    Evaluates the 3D magnetic field via a 2D poloidal 6th order polynomial and
+    1D toroidal 4th order Bezier spline.
+    
+Nek5000
+    Evaluates the 3D vector field using `Nek5000 <https://nek5000.mcs.anl.gov/>`_
+    spectral elements.
```

Comment:
Might need to have verbiage here about whether the particular instance of VisIt the user is using has this feature enabled (and I honestly don't know if that situation is more complex for client/server)

Reply:
Perhaps we should also detect if the user has these third party libraries installed and only display these options then.